src/resinator/preprocess.zig

```diff
@@ -67,7 +68,7 @@ pub fn appendClangArgs(arena: Allocator, argv: *std.ArrayList([]const u8), optio
     }
 
     if (!options.ignore_include_env_var) {
-        const INCLUDE = std.process.getEnvVarOwned(arena, "INCLUDE") catch "";
+        const INCLUDE = (introspect.EnvVar.INCLUDE.get(arena) catch @panic("OOM")) orelse "";
```

Comment:
If this environment variable was guaranteed to only be read during `zig rc` and never any other `zig`  command, would this introspect code still be needed in the resinator code?

Reply:
No, I think if that guarantee was upheld, then the resinator patches here could be reverted :+1: